azure_functions_worker/dispatcher.py

```diff
@@ -297,13 +297,8 @@ async def _handle__worker_init_request(self, request):
             DependencyManager.prioritize_customer_dependencies()
 
         if DependencyManager.is_in_linux_consumption():
-            logger.info(
-                "Importing azure functions in WorkerInitRequest")
             import azure.functions  # NoQA
 
-        if CUSTOMER_PACKAGES_PATH not in sys.path:
```

Comment:
Wasn't this fixed in an earlier PR? Why remove this now?

Reply:
We don't actually need this check. Doesn't give much info.
Also even though this was fixed, it won't show up in prod logs but locally it will when using core tools. Customers get this warning message while running func host start. Better we remove it